Jenkinsfile

```diff
@@ -26,21 +26,41 @@ pipeline {
       }
     }
 
-    stage('Adminrouter') {
-      steps {
-        script {
-          task_wrapper('mesos-sec', master_branches, '8b793652-f26a-422f-a9ba-0d1e47eb9d89', '#dcos-security-ci') {
-              stage('Cleanup workspace') {
-                  deleteDir()
-              }
-          
-              stage('Checkout') {
-                  checkout scm
-              }
-          
-              load 'Jenkinsfile-insecure.groovy'
-	    }
-        }  
+    stage('Build') {
+      parallel {
+        stage('Tox') {
+	  agent {
+	    label 'python-dind'
+	  }
+	  steps {
+	    sh('rm -rf dcos-release.config.yaml')
```

Comment:
How about moving it into Makefile and run in docker container so it will be easier to run tox locally on commit hook?

Reply:
Thanks for resurrecting this. Yes, that's a good idea. But we need the credentials first. I created a ticket https://jira.d2iq.com/browse/D2IQ-66808

Reply:
I'll do it as part of #6818